docs/ZendeskMultiObjects-batchsource.md

```diff
@@ -53,22 +55,25 @@ the specified date. The date must be provided in the date format:
 |                                  | YYYY-MM-DDThh:mm:ss-hh:mm | 1999-01-01T23:01:01-08:00 |
 |                                  | YYYY-MM-DDThh:mm:ssZ      | 1999-01-01T23:01:01Z      |
 
-Specifying this along with `Start Date` allows reading data modified within a specific time window. 
+If you enter an End Date and Start Date, the data is modified within a specific time window. 
 If no value is provided, no upper bound is applied.
 
-**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records which have Zendesk score
-equal to the specified score.
+**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records that have Zendesk score equal to the specified score. 
+Only Applicable for Satisfaction Ratings object.
 
 **Table Name Field:** The name of the field that holds the table name. Must not be the name of any table column that
 will be read. Defaults to `tablename`.
 
 ### Advanced
 
-**Max Retry Count:** Maximum number of retry attempts.
+**Max Retry Count:** Maximum number of retry attempts. Maximum number of retry attempts. Default is 20.
```

Comment:
you have Maximum number of retry attems 2x. Delete one.

Reply:
done